hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterRpc.java

```diff
@@ -1450,6 +1452,95 @@ public void testProxyRestoreFailedStorage() throws Exception {
     assertEquals(nnSuccess, routerSuccess);
   }
 
+  @Test
+  public void testRewnewLease() throws Exception {
```

Comment:
This test is has become little big, Can we split the create & append apart into different tests? Can extract the common stuff into a util method and reuse